larpandoracontent/LArControlFlow/MasterAlgorithm.cc

```diff
@@ -670,6 +670,7 @@ StatusCode MasterAlgorithm::Copy(const Pandora *const pPandora, const CaloHit *c
     // ATTN Parent of calo hit in worker is corresponding calo hit in master
     parameters.m_pParentAddress = static_cast<const void*>(pCaloHit);
     parameters.m_larTPCVolumeId = pLArCaloHit->GetLArTPCVolumeId();
+    parameters.m_daughterVolumeId = pLArCaloHit->GetDaughterVolumeId();
```

Comment:
Looks like this change is a subset of Maria's changes, so can pick Maria's version in any merge.